kolibri/core/assets/src/core-actions.js

```diff
@@ -54,9 +126,214 @@ function togglemodal(store, bool) {
   }
 }
 
+
+/**
+ * Create models to store logging information
+ * To be called on page load for content renderers
+ */
+function initContentSession(store, Kolibri, channelId, contentId, contentKind) {
```

Comment:
@richard We should be able to just extract the channel_id, content_id, and content_kind of the currently viewed content from vuex -- I imagine that's preferable to having the .vue component need to pass them in here?